test/options/quotes.spec.ts

```diff
@@ -23,7 +23,7 @@ test(`should successfully beautify JavaScript text with single quotes`, () => {
 test(`should successfully beautify JavaScript text with double quotes`, () => {
   const unibeautify = newUnibeautify();
   unibeautify.loadBeautifier(beautifier);
-  const quote = '"';
+  const quote = "'";
```

Comment:
This is a bug with one of the beautifiers. Maybe Prettier or Pretty Diff.

Reply:
.unibeautifyrc.yml has `quotes: "double"`, so I believe its beautifying as designed.  Can Unibeautify CI be configured to ignore specific rules for specific directories?

Reply:
Not yet. Eventually will though.

Reply:
It is a Pretty Diff bug.

Prettyier: http://playground.unibeautify.com/#N4IgNghgdg5grhGBTAchAtkkAuEApCANwgGUBjAJwEsAHAFxABoQB7eqlqAZx1AOPLV6vEACMkEOHSoAzKkgo9sAbRAAFCkjrSFIALrMIYKjCgB9CFy4momKHSUyjXJIYoUWAdzM0Im7jgglgC0UEhIACaRTGKaEADWZmQAFhBUYRFmmHTJLBFKdBRwriBIUJmeVDlJLOjoEDiFxcxlFVXJZi7oVGQsYJyNRSXpUfZJqRSBAAQxI2V0nVQAXljYAEzMAFZcAB5mohQQZPFajs4l6HBg0sZhZnQKUH4AnoPNrKKbSGQLZHAUYGenV8ZHSMDeJQAjnAWA8lCAIiw4KIwFhmFwQUgLDIHhQLFBOGYZHAoD8OFAcE4wC5mJ5DjQzLcsajYDkcAAOAAMAF9ucwWNQYOkjAAVJA7Bi4Ymk6ScKbJJBgfoAdQFYAiAAoAJRTYAAHVJnC4fSQADp+jANQByPUgK1agDcBu5IG5QA
Pretty Diff: http://playground.unibeautify.com/#N4IgNghgdg5grhGBTAchAtkkAuEApCANwgGUBjAJwEsAHAFxABoQB7eqlqAZx1AOPLV6vEACMkEOHSoAzKkgo9sAbRAAFCkjp0AngAIAIrJkgAuswhgqMKAH0IXLtaiYodJTMtckFihRYA7rY0EJrcOCAOALRQSEgAJglMYpoQANa2ZAAWEFSx8baYdFks8Up0FHA+IEhQBQFUxZks6OgQOBVVzLX1jVm23uhUZCxgnB2V1XmJbpk5FBF6ydO1dANUAF5Y2ABMzABWXAAetqIUEGRpWh5e1ehwYNJWsbZ0ClChOhNdrKL7SGQ1mQ4BQwDoBiEyHkYN9qgBHOAsN5KEDxFhwURgLDMLiQpD2GRvCj2KCcWwyOBQQEcKA4TxgbzMALnGi2Z74rGwYo4AAcAAYAL4C5gsagwPKWAAqSCODFwFKp0k4eiySDAYwA6qKwPEABQASj0wAAOlTOFxRkgAHRjGC6gDkxpA9v1AG5TQKQAKgA

Reply:
Made a bug report: https://github.com/prettydiff/prettydiff/issues/508